src/jobs/publish.yml

```diff
@@ -147,6 +161,15 @@ steps:
         - setup_remote_docker:
             docker_layer_caching: <<parameters.remote-docker-dlc>>
 
+  - when:
+      condition: <<parameters.use-workspace>>
+      parameters:
```

Comment:
No parameters should be defined here.

Reply:
Does this lint properly? If this works, I assume it is being ignored.

Reply:
It seems that it passes your linting tests but you are right, I don't know exactly why parameters were added here as well.